### DISCUSSION TITLE
What if illegal content is pushed to the blockchain?

### DISCUSSION THREAD

**Discussion by panzi** [nestdotland/nest.land#327] [Q&A]
What if illegal content is pushed to the blockchain? Like a copyright violation, documented child abuse, hate speech, etc.? You will be required by law to remove those. How do you do that if the blockchain is immutable? Flagging it is irrelevant.

**Comment by maximousblk** [comment]
in case such activity is reported, the most we could do depending on the situation is remove the reference to the artifact from our database to make sure it's not distributed through nest. we can't remove it from the blockchain. you can ask for further info regarding this on the [Arweave Dev Discord](https://discord.gg/BXk8tq7).

**Reply by panzi** [reply]
I don't think copyright lawyers and law enforcement agencies will like that answer. Just want to make you think about what to do when those come knocking.

**Reply by littledivy** [reply]
nest.land does not store content on its servers. Everything is uploaded straight to Arweave. We can only remove links to the content but the data is not under our control.  Any concerned authorities should reach out to Arweave instead.

**Reply by panzi** [reply]
I see.
